### PR TITLE
fix: support for hardcore on the server browser

### PR DESCRIPTION
**Pull request by JariKCoding**
Added support for the hardcore skull icon in the server browser
image

### CODE REVIEW

File: src/client/component/getinfo.cpp [hunk 1]
```diff
@@ -112,7 +112,8 @@ namespace getinfo
 				info.set("playmode", utils::string::va("%i", game::Com_SessionMode_GetMode()));
 				info.set("gamemode", utils::string::va("%i", Com_SessionMode_GetGameMode()));
 				info.set("sv_running", utils::string::va("%i", game::is_server_running()));
-				info.set("dedicated", utils::string::va("%i", game::is_server() ? 1 : 0));
+				info.set("dedicated", utils::string::va("%i", game::is_server() ? 1 : 0));
+				info.set("hc", utils::string::va("%i", game::Com_GametypeSettings_GetUInt("hardcoremode", false)));
```
Comment (diamante0018) [inline src/client/component/getinfo.cpp:116]: now that i think of it, why is utils::string::va being used this much in this file when std::to_string exists? 😂 
probably a question for another day, patch looks good.

**Review by diamante0018 — CHANGES_REQUESTED** [review]

**Review by diamante0018 — APPROVED** [review]

File: src/client/component/getinfo.cpp (hunk 1 above)
Comment (momo5502) [reply src/client/component/getinfo.cpp]: that really screams like good old c&p 😂

**Comment by diamante0018** [comment]
@momo5502 could u merge this so i can then make a new pr for ::va?

**Comment by momo5502** [comment]
> @momo5502 could u merge this so i can then make a new pr for ::va?
> Or you wanted myself or jarik do address it right away?

you uave to close open comments i think. cant see it on my phone

**Pull request merged**